frontend/src/Buildings/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  View, Text, ScrollView,
+  StyleSheet, ActivityIndicator,
+} from 'react-native';
+// import { CardView } from '../common/components';
+import BuildingItem from './buildingItem';
+import { fetchBuildings } from './actionCreator';
+import townhallIcon from '../../assets/buildings/townhall.png';
+import academyIcon from '../../assets/buildings/academy.png';
+import farmIcon from '../../assets/buildings/factory.png';
+import mineIcon from '../../assets/buildings/mine.png';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  horizontal: {
+    flexDirection: 'row',
+    justifyContent: 'space-around',
+    padding: 10,
+  },
+  scrollviewStyle: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'space-between',
+  },
+});
+
+function Buildings() {
+  const buildingsComponent = useSelector((state) => state.buildings.buildingsInfo);
```

Comment:
listOfBuildings maybe would be a better choice? or myBuildings? or something? it is not a component